sdks/go/sdk.go

```diff
@@ -42,6 +43,14 @@ type SDK struct {
 	alpha  *Alpha
 }
 
+// ErrorLog is a function to log the error.
+type ErrorLog func(error, string)
+
+// Logger is a pluggable function that outputs the error message to standard error.
+var Logger ErrorLog = func(err error, msg string) {
```

Comment:
I feel like error should come second - like it would in a printf statement. WDYT?

```suggestion
var Logger ErrorLog = func(msg string, err error) {
```

Reply:
Swapped the positions